main.go

```diff
@@ -422,7 +422,17 @@ func run() (err error) {
 	return
 }
 
+func handlePanic() {
+	// detect if panic occurred or not
```

Comment:
Please fix the comment - "Detect if panic happens in main go routine."

Reply:
Fixed